llvm/test/CodeGen/RISCV/GlobalISel/legalizer/legalize-zfa-f16.ll

```diff
@@ -0,0 +1,78 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
```

Comment:
This tests more than the legalizer. Put it in the llvm/test/CodeGen/RISCV/GlobalISel/ directory and name it half-zfa.ll like the SelectionDAG test.

Reply:
Moved this I temporarly disabled the Fmaximum and Fminimum tests for consistency I think we can still test the lib call cases for s32 and s64 cases but I need to figure out the libcall pattern for it